src/components/shared/datePicker/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import DatePicker from 'react-native-date-picker';
+
+interface DatePickerInputProps {
+  date: Date;
+  open: boolean;
+  mode?: 'date' | 'time' | 'datetime' | undefined;
```

Comment:
If you set a prop like optional, automatically is set like undefined

Reply:
I deleted this component, we directly import the datepicker from the library